src/input/Search.js

```diff
@@ -18,8 +18,18 @@ class Search extends Component {
   }
 
   clearText() {
-    const ref = this.props.textInputRef;
-    this.refs[ref].clear();
+    if (this.props.onChangeText) {
+      this.props.onChangeText('');
```

Comment:
I need onChangeText called after text is cleared so tha my onChangeText function can reload my data. I might just not show my clearIcon until this is merged.